src/components/detail-tags/_detail-tags.scss

```diff
@@ -0,0 +1,54 @@
+.detail-tags {
+  display: flex;
+  flex-wrap: wrap;
+  position: relative;
+  width: 100%;
+
+  &--hx {
+    @media screen and (min-width: map-get($breakpoints, 'breakpoint-4')) {
+      bottom: $spacing-8;
+      position: absolute;
+      justify-content: flex-end;
+      right: $spacing-8;
+      width: 18rem;
+    }
+  }
+
+  &--rift {
+    margin-top: $spacing-9;
+  }
+
+  &__heading {
+    color: var(--detail-heading);
+    font-family: $font-family-sans-serif-extrabold;
+    font-size: .75rem;
+    letter-spacing: .1rem;
+    position: absolute;
+    text-transform: uppercase;
+    top: -1.5rem;
+  }
+  
+  &__item {
+    align-items: center;
+    background-color: var(--color-detail-bg);
+    border-radius: $border-radius-sm;
+    display: flex;
+    margin: .25rem;
+    padding: .4rem;
+  }
+
+  &__icon {
+    align-items: center;
+    display: flex;
+    justify-content: center;
+    height: .8rem;
+    margin-right: .3rem;
```

Comment:
Do we have a spacing variable for this? If not, can it use one we do have or not?